Dockerfile.ocp

```diff
@@ -14,7 +14,7 @@ LABEL io.k8s.display-name="OpenShift Prometheus" \
       io.k8s.description="The Prometheus monitoring system and time series database." \
       io.openshift.tags="prometheus,monitoring" \
       maintainer="OpenShift Development <dev@lists.openshift.redhat.com>" \
-      version="v2.10.0"
+      version="v2.14.0"
```

Comment:
Good catch, I guess this was missed in the last upgrade.

Reply:
Yeah I don't think it matters that much TBH.